airbyte-integrations/connectors/source-lever-hiring/unit_tests/test_source.py

```diff
@@ -10,7 +10,8 @@
 
 def setup_responses():
     responses.add(
-        responses.POST, "https://sandbox-lever.auth0.com/oauth/token", json={"access_token": "fake_access_token", "expires_in": 3600}
+        responses.POST, "https://sandbox-lever.auth0.com/oauth/token", json={"access_token": "fake_access_token", "expires_in": 3600},
+        responses.GET, "https://api.lever.co/v1/opportunities", json={"api_key": "fake_api_key", "expires_in": 3600}
```

Comment:
thanks for adding another mock response here. However, in order to use it, you may adjust `test_config` so that the newly-added response will be returned. Test_config is defined under [conftest.py](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connectors/source-lever-hiring/unit_tests/conftest.py#L9). 

Just for your reference, you can use [pytest parametrize](https://docs.pytest.org/en/6.2.x/parametrize.html) to set up the config or add another `test_check_connection` method.

Reply:
Done as well, or at least tried to - I don't understand everything about the way those tests are organized when comparing it to other connectors where conftest.py may not exist. Let me know if that would work in your opinion?

Reply:
yes, it works. Thanks for addressing the unit tests.